symmetric-core/src/main/java/org/jumpmind/symmetric/io/data/transform/BshColumnTransform.java

```diff
@@ -50,7 +49,8 @@
 import bsh.Interpreter;
 import bsh.TargetError;
 
-public class BshColumnTransform implements ISingleNewAndOldValueColumnTransform, IBuiltInExtensionPoint {
+public class BshColumnTransform extends AbstractColumnTransform implements ISingleNewAndOldValueColumnTransform 
+{
```

Comment:
I think the IBuiltInExtensionPoint interface is needed on all these transforms because they are built-in extensions, not ones written by user.  Without that interface, it will start logging about these extension points and they show up on the About -> Extensions screen as provided by the user.